glide.yaml

```diff
@@ -32,3 +32,5 @@ import:
   - util/retry
 - package: github.com/vishvananda/netns
 - package: github.com/pkg/errors
+- package: github.com/mattn/goveralls
+  version: ^0.0.2
```

Comment:
Looking commit history, https://github.com/mattn/goveralls/commits/master, after v0.0.2, there are several commits.
In addition, v0.0.2 is released at Nov 2017, so roughly no release for 1year even though master is updated.

So I'm wondering we use master instead of release...
I suppose we just keep in mind for now and we use v0.0.2 first.

Reply:
Ahh shoot, this is a mistake! This shouldn't be here. Thanks Tomo, it's unnecessary.